applicationinsights/data-plane/src/test/java/com/microsoft/azure/applicationinsights/query/ApplicationInsightsDataClientTests.java

```diff
@@ -58,6 +58,8 @@ public void canQuery() {
     public void canGetMetric() {
         MetricsResult metricResult = applicationInsightsClient.metrics().get(appId, MetricId.AVAILABILITY_RESULTSAVAILABILITY_PERCENTAGE);
         // Validate properties
+        Assert.assertNotNull(metricResult.value().additionalProperties());
```

Comment:
Seems like the client doesn't properly populate this field from JSON? Here's what the return shape from API looks like, but the assert fails. I would expect something like `metricResult.value().additionalProperties().get("availabilityResults/availabilityPercentage")` should work, but `metricResult.value().additionalProperties()` is null.

```json
{
  "value": {
    "start": "2018-09-10T07:47:08.518Z", 
    "end":"2018-09-10T19:47:08.518Z",
    "availabilityResults/availabilityPercentage": {
      "avg":100
    }
  }
}
```